### DISCUSSION TITLE
NewsLazer Icon

### DISCUSSION THREAD

**Discussion by Fskum** [elrumo/macOS_Big_Sur_icons_replacements#1311] [Icon request]
Hey, I'm looking for a BigSur icon for the NewsLazer application. They updated their application for Mac in June of this year, but they chose not to update their icon. Is there anyone who will try to make one, then I would be very grateful.

Their logo can be seen here https://newslazer.com I imagine something with a lion and something laser, but I look forward to seeing what can be done.

Thanks in advance
![newslazer](https://user-images.githubusercontent.com/86257880/138748864-49a25e80-0669-4dc7-8689-d528bf20c508.jpg)

**Comment by cake03** [comment]
<img alt='Newslazer' src='https://user-images.githubusercontent.com/80169337/140872085-341aa79e-5ce3-4010-b398-2c552d9fb34b.png' height='512' />

**Reply by Fskum** [reply]
Really nice, I love that the icon got a black background, thought all the app icons are white today.. 1000 thanks for the great work, I really appreciate it ❤️
